static/js/place/main.tsx

```diff
@@ -81,7 +87,7 @@ class MainPane extends React.Component<MainPanePropType> {
     const currentPageTopic = this.props.topic;
     const isOverview = currentPageTopic === "Overview";
     return (
-      <>
+      <RawIntlProvider value={this.props.intl}>
```

Comment:
can we move this to the place.ts module, since it pertains to app setup?

Reply:
Yes let's do that once place.ts is tsx'ed